lms/djangoapps/discussion_api/api.py

```diff
@@ -69,10 +83,16 @@ def get_module_sort_key(module):
         {
             "id": None,
             "name": category,
+            "thread_list_url": get_thread_list_url(
+                request,
+                course_key,
+                [item.discussion_id for item in sorted(modules_by_category[category], key=get_module_sort_key)]
```

Comment:
Factor out `sorted(modules_by_category[category])`, which is also used a few lines later, to a variable.